test/command-line.js

```diff
@@ -41,3 +41,54 @@ describe('command line', function () {
     });
   });
 });
+
+describe('command line watch mode', function () {
+  var watchProc;
+  var stdout = '';
+  after(function() {
+    watchProc.kill('SIGINT');
+  });
+  it('jade --no-debug --client --name-after-file --watch input-file.jade (pass 1)', function (done) {
+    fs.writeFileSync(__dirname + '/temp/input-file.jade', '.foo bar');
+    fs.writeFileSync(__dirname + '/temp/input-file.js', 'throw new Error("output not written (pass 1)");');
+    var args = [__dirname + '/../bin/jade.js', '--no-debug', '--client', '--name-after-file', '--watch', 'input-file.jade']
+    watchProc = cp.spawn('node', args,  {
+      cwd: __dirname + '/temp'
+    });
+
+    watchProc.stdout.setEncoding('utf8');
+    watchProc.on('error', function(err) {
```

Comment:
Did passing in `done` as the callback not work here?  I thought it was much cleaner that way.